test/test_decoding.py

```diff
@@ -0,0 +1,144 @@
+import tensorflow as tf
+
+from wavenet import WaveNet
+from test_encoding import manual_encode
+import json
+import numpy as np
+
+def manual_decode(signal, channels):
+    # Calculate inverse mu-law companding and dequantization
+    mu = channels
+    output = signal.astype(np.float32)
+    y = (2*output - 1) / mu
```

Comment:
Shouldn't this be `2 * (output / mu) - 1`?
It was recently fixed in this commit: https://github.com/ibab/tensorflow-wavenet/commit/9f85246631d9ffc3d76ae881442c993a5f6e91b6
I think the reason travis is failing is that there's a mismatch between the manual decoding and what's now on `master`.